apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/page/service/PageHistoryFinder.java

```diff
@@ -0,0 +1,58 @@
+package gov.cdc.nbs.questionbank.page.service;
+
+import gov.cdc.nbs.questionbank.page.model.PageHistory;
+import lombok.RequiredArgsConstructor;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class PageHistoryFinder {
+
+  private final JdbcTemplate jdbcTemplate;
+
+  private static final String QUERY = """
+      with target as (
+                       SELECT
+                           template_nm as template_name
+                       FROM WA_template
+                       WHERE wa_template_uid = ?
+                   )
+      SELECT
+        hist.publish_version_nbr AS publishVersionNbr,
+        CONVERT(varchar, hist.last_chg_time, 101) AS lastUpdatedDate,
+        userProfile.first_nm || ' ' || userProfile.last_nm AS lastUpdatedBy,
```

Comment:
Issue: the `||` syntax is not supported in MSSQL. API fails to return any data due to `com.microsoft.sqlserver.jdbc.SQLServerException`